New functions to add user(s) to group(s)

vanatteveldt opened the pull request:
I've added two functions to add users to a group from R. This is useful if you have e.g. a spreadsheet of desired group enrollments.

I've tried to follow your coding style, but please let me know if you would like to see any changes. Also, I can add an example in the README if you want?

daranzolin commented:
An important and necessary addition! Everything looks good. I need to revamp the vignette sometime soon.

If you want to add an example to the README and also add yourself as a contributor to the DESCRIPTION, I'll happily merge the request.

daranzolin commented:
I also adjusted `canvas_query` to authenticate in the header, so you no longer need to pass `access_token = check_token()` to the `args` object.

vanatteveldt commented:
@daranzolin I've made some changes:
- Renamed add_multie_groups to add_groups and un-exported add_group. I think it's more idiomatic to assume everything operates on a vector in R.
- Added a new function add_groups
- Added examples
- Added myself to the contributors

ucscout commented:
Please unsubscribe *info@ucscout.org <info@ucscout.org>* from this thread.
David no longer works for Scout.

Thank you,

Scout Team

On Thu, Mar 22, 2018 at 12:51 PM, Wouter van Atteveldt <
notifications@github.com> wrote:

> @daranzolin <https://github.com/daranzolin> I've made some changes:
>
>    - Renamed add_multie_groups to add_groups and un-exported add_group. I
>    think it's more idiomatic to assume everything operates on a vector in R.
>    - Added a new function add_groups
>    - Added examples
>    - Added myself to the contributors
>
> —
> You are receiving this because you are subscribed to this thread.
> Reply to this email directly, view it on GitHub
> <https://github.com/daranzolin/rcanvas/pull/15#issuecomment-375435900>,
> or mute the thread
> <https://github.com/notifications/unsubscribe-auth/APP9RQWZ0FmRqunEbmlaH_BhvO6lrFgSks5thADEgaJpZM4PMEvt>
> .
>



-- 
Scout from University of California
408.450.4962
info@ucscout.org

daranzolin commented:
Looks great!

vanatteveldt commented:
Cool, thanks!
